jelly/src/main/java/org/kohsuke/stapler/jelly/JellyClassLoaderTearOff.java

```diff
@@ -56,46 +55,43 @@ public JellyClassLoaderTearOff(MetaClassLoader owner) {
     }
 
     public TagLibrary getTagLibrary(String nsUri) {
-        LoadingCache<String,TagLibrary> m=null;
+        ConcurrentMap<String,TagLibrary> m=null;
         if(taglibs!=null)
             m = taglibs.get();
         if(m==null) {
-            m = CacheBuilder.newBuilder().build(new CacheLoader<String,TagLibrary>() {
-                public TagLibrary load(String nsUri) {
+            m = new ConcurrentHashMap<>();
+            taglibs = new WeakReference<>(m);
+        }
+        TagLibrary tl = m.computeIfAbsent(nsUri, key -> {
                     if(owner.parent!=null) {
                         // parent first
-                        TagLibrary tl = owner.parent.loadTearOff(JellyClassLoaderTearOff.class).getTagLibrary(nsUri);
-                        if(tl!=null)    return tl;
+                        TagLibrary taglib = owner.parent.loadTearOff(JellyClassLoaderTearOff.class).getTagLibrary(key);
+                        if(taglib!=null)    return taglib;
```

Comment:
Avoid variable renaming not directly needed, to minimize patch.

Reply:
It _is_ directly needed to avoid a compilation failure:

```
[ERROR] stapler/stapler/jelly/src/main/java/org/kohsuke/stapler/jelly/JellyClassLoaderTearOff.java:[68,36] variable tl is already defined in method getTagLibrary(java.lang.String)
```